packages/ui/src/components/va-switch/VaSwitch.vue

```diff
@@ -121,6 +124,9 @@ export default defineComponent({
     const { hasKeyboardFocus, keyboardFocusListeners } = useKeyboardOnlyFocus()
     const { isChecked, computedError, isIndeterminate, ...selectable } = useSelectable(props, emit, elements)
 
+    const computedBackground = computed(() => isChecked.value ? getColor(props.color) : getColor('gray'))
```

Comment:
Not related to this task, but it will be nice to have this `'gray'` color in props. Like, `props.offColor` or `disabledColor`. Feel free to update props. Thanks.